src/mlpack/tests/distribution_test.cpp

```diff
@@ -15,6 +15,7 @@
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
 #include <mlpack/core.hpp>
+#include <mlpack/core/math/ccov.hpp>
```

Comment:
I would say, let's include `ccov.hpp` in `mlpack/prereqs.hpp` because it would have previously been included through `arma_extend/`.

Reply:
Added to `mlpack/prereqs.hpp`